model_templates/training/python3_sklearn_binary_schema_validation/README.md

```diff
@@ -0,0 +1,48 @@
+## Advanced Python Template Model
```

Comment:
Doesn't have to be in this PR, but you should make a ticket to write a readme for all the possible validation params, maybe have them commented out in the model-metadata.yaml

Reply:
https://datarobot.atlassian.net/browse/RAPTOR-5286